speechbrain/inference/ASR.py

```diff
@@ -787,7 +787,7 @@ def encode_chunk(
     @torch.no_grad()
     def decode_chunk(
         self, context: ASRStreamingContext, x: torch.Tensor
-    ) -> tuple[list, list]:
+    ) -> Tuple[List[str], List[int]]:
```

Comment:
Would be incorrect, it should rather be
```suggestion
    ) -> Tuple[List[str], List[List[int]]]:
```
As it's potentially a batch of tokens, not just a single batch.